doc/interfaces/reconstruction_flow.rst

```diff
@@ -0,0 +1,269 @@
+
+    mkdir recons_csa_output
+
+The workflow for the CSA reconstruction method requires the paths to the
+diffusion input file, b-values file, b-vectors file and mask file. The optional
+arguments can also be provided. In this case, we will be specifying whether or
+not to save pam volumes as single nifti files (``extract_pam_values``) and the
+output directory (``out_dir``).
+
+To perform the CSA reconstruction, we will run the command as::
+
+    dipy_fit_csa data/stanford_hardi/HARDI150.nii.gz data/stanford_hardi/HARDI150.bval data/stanford_hardi/HARDI150.bvec stanford_hardi_mask/brain_mask.nii.gz --extract_pam_values --out_dir "recons_csa_output"
+
+This command will save the CSA metrics to the specified output directory.
+
+-----------------------------------
+Intravoxel incoherent motion (IVIM)
+-----------------------------------
+
+The intravoxel incoherent motion (IVIM) model describes diffusion
+and perfusion in the signal acquired with a diffusion MRI sequence
+that contains multiple low b-values. The IVIM model can be understood
+as an adaptation of the work of Stejskal and Tanner [Stejskal65]_
+in biological tissue, and was proposed by Le Bihan [LeBihan84]_.
+The model assumes two compartments: a slow moving compartment,
+where particles diffuse in a Brownian fashion as a consequence of thermal
+energy, and a fast moving compartment (the vascular compartment), where
+blood moves as a consequence of a pressure gradient. In the first compartment,
+the diffusion coefficient is $\mathbf{D}$ while in the second compartment, a
+pseudo diffusion term $\mathbf{D^*}$ is introduced that describes the
+displacement of the blood elements in an assumed randomly laid out vascular
+network, at the macroscopic level. According to [LeBihan84]_,
+$\mathbf{D^*}$ is greater than $\mathbf{D}$.
+
+We are using ``ivim`` dataset. You can also use your own data.
+
+Create the directories in which to save the output volumes (e.g.: ``recons_ivim_output``)
+and mask file (e.g.: ``ivim_mask``)::
+
+    mkdir recons_ivim_output ivim_mask
+
+In order to run the IVIM reconstruction method, we need to specify the
+locations of the diffusion input file, b-values file, b-vectors file and
+mask file followed by the optional arguments. In this case, we will be 
+specifying the value to split the bvals to estimate D for the two-stage
+process of fitting (``split_b_D``) and the output directory (``out_dir``).
+
+To get the mask file, we will run the following command::
+
+    dipy_median_otsu data/ivim/HARDI150.nii.gz --vol_idx 3 4 5 6 --out_dir "ivim_mask"
+
+Then, to perform the IVIM reconstruction we will run the command as::
+
+    dipy_fit_ivim data/ivim/ivim.nii.gz data/ivim/ivim.nii.gz.bval data/ivim/ivim.nii.gz.bvec ivim_mask/brain_mask.nii.gz --split_b_D 250 --out_dir "recons_ivim_output"
+
+This command will save the IVIM metrics to the directory ``recons_ivim_output``.
+
+In case, you did not specify the output directory, the output volumes
```

Comment:
In case the output directory was not specified, the (...)

Reply:
Reworded in 4ce067c